mars/services/task/execution/ray/tests/test_ray_execution_backend.py

```diff
@@ -195,7 +195,8 @@ def fake_init(self):
         pass
 
     with mock.patch.object(ThreadedServiceContext, "__init__", new=fake_init):
-        context = RayExecutionContext({"abc": o}, {}, None)
+        config = RayExecutionConfig.from_execution_config({"backend": "mars"})
```

Comment:
Why passing `mars` to `RayExecutionConfig`

Reply:
It's a mock config, I will fix it.